files.js

```diff
@@ -74,7 +77,6 @@ router.post('/files', function(req, res, next) {
                        'Filename not provided.'));
 });
 
-// TODO: do we need to allow crossdomain and method override?
 router.post('/files/:filename',
```

Comment:
The best method is to just pass `middlewares.allowCrossDomain` here in the route.

```
router.post('/files/:filename',
                  middlewares.allowCrossDomain,
                  bodyParser.raw({type: '*/*', limit: '20mb'}),
                  middlewares.handleParseHeaders,
                  processCreate);
```